helpers/session.js

```diff
@@ -55,7 +55,21 @@ module.exports = class Session extends Helper {
     return new Promise((resolve, reject) => {
       this._getSession()
         .then(sessionData => {
-          resolve(sessionData[`${SESSION_KEY_PREFIX}-${sessionKey}`] || {});
+          let key;
+          if (!sessionKey) {
+            const keys = Object.keys(sessionData);
+            if (keys.length === 3) {
```

Comment:
It may be clearer if the filtering is done first, then check the length === 1 - I'll do this